subcommand/connect-init/command.go

```diff
@@ -51,6 +53,8 @@ func (c *Command) init() {
 	c.flagSet.StringVar(&c.flagACLAuthMethod, "acl-auth-method", "", "Name of the auth method to login to.")
 	c.flagSet.StringVar(&c.flagPodName, "pod-name", "", "Name of the pod.")
 	c.flagSet.StringVar(&c.flagPodNamespace, "pod-namespace", "", "Name of the pod namespace.")
+	c.flagSet.StringVar(&c.flagAuthMethodNamespace, "auth-method-namespace", "", "Consul namespace the auth-method is defined in")
+	c.flagSet.StringVar(&c.flagServiceNamespace, "service-namespace", "", "Consul destination namespace of the service.")
```

Comment:
Now looking at it in combination with `pod-namespace` I wonder if we need to disambiguate this and be explicit that this the Consul namespace? maybe `consul-service-namespace`?